packages/a2a-server/src/persistence/gcs.test.ts

```diff
@@ -227,6 +227,35 @@ describe('GCSTaskStore', () => {
         'tar.c command failed to create',
       );
     });
+
+    it('should throw an error if taskId contains path traversal sequences', async () => {
+      const store = new GCSTaskStore('test-bucket');
+      const maliciousTask: SDKTask = {
+        id: '../../../malicious-task',
```

Comment:
GCS doesn't actually support path traversal as it simply has a flat namespace. But it's still not great for us to accept arbitrary inputs here.